.github/workflows/tests.yml

```diff
@@ -435,7 +430,7 @@ jobs:
     name: Integration tests / Elixir
     runs-on: ubuntu-latest
     needs: [setup, file_changes, build_app]
-    if: ${{ needs.file_changes.outputs.ex }}
+    if: ${{ !(contains(github.head_ref, 'dependabot') || github.actor == 'dependabot[bot]') && needs.file_changes.outputs.ex }}
```

Comment:
Do you need the new case? It looks like `needs.file_changes.outputs` will all be false because that step also has the "not a dependabot PR" check.

Reply:
removing the case seems to work great! thanks